community-content/pytorch_pre_built_images_deployment/pytorch-text-classification-vertex-ai-deploy.ipynb

```diff
@@ -0,0 +1,1658 @@
+{
```

Comment:
<p>what is meant by <em>[Optional]</em></p><p> </p><p>we -&gt; you </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
It's actually optional (return 0/1 or positive/negative). To be easy to understand this sample without any other changes by users, keep it necessary.